source/FreeRTOS_DHCP.c

```diff
@@ -887,6 +909,7 @@
                 xAddress.sin_port = ( uint16_t ) dhcpCLIENT_PORT_IPv4;
```

Comment:
I think it wouldn't hurt if all fields of `xAddress` will be set, as is also done in `prvCreateDHCPv6Socket()`.
~~~diff
+    memset( &xAddress, 0, sizeof( xAddress ) );
+    xAddress.sin_family = FREERTOS_AF_INET4;
+    xAddress.sin_len = ( uint8_t ) sizeof( xAddress );
     xAddress.sin_port = ( uint16_t ) dhcpCLIENT_PORT_IPv4;
~~~

Reply:
done

Reply:
> done

Thank you @bjsowa for the PR. It's good work and very much appreciated. I already approved it yesterday.
Hein